notebooks/02_foundations__connection_riemannian_metric.ipynb

```diff
@@ -0,0 +1,2266 @@
+{
```

Comment:
<p>NIT: to gain intuition on?</p><p> </p><p>Typo: we know that a geodesic is the shortest path between two points in space.</p><p>Typo: through curved spacetime.</p><p> </p><p>Nice physics explanation!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/1551/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
<p>Thank you! I ended up re-writing this because it didnt sound as good the second time i read it haha</p>